frontend/packages/client/src/contexts/Web3.js

```diff
@@ -71,6 +73,7 @@ export function Web3Provider({ children, network = 'testnet', ...props }) {
       'app.detail.icon': iconUrl,
       'accessNode.api': accessApi, // connect to Flow
       'discovery.wallet': walletDiscovery, // use wallets on public discovery
+      'discovery.wallet.method': 'POP/RPC',
       'discovery.authn.endpoint': walletDiscoveryApi, // public discovery api endpoint
       'discovery.authn.include': walletDiscoveryInclude, // opt-in wallets
     });
```

Comment:
on [this line](https://github.com/DapperCollectives/CAST/pull/334/files#diff-baedbdacd01ccf4a2352e300c4f7c4e4a22f88475838b4a1676079357567e915R113 )
you can add: 
```
 {
          f_type: 'Service',
          f_vsn: '1.0.0',
          method: 'IFRAME/RPC',
          provider: {
            icon: '/images/dapper.svg',
            name: 'Dapper Wallet',
          },
          uid: 'dapper-wallet#authn',
        },
````
to enable local dev